assets/controllers/form-related-links_controller.js

```diff
@@ -0,0 +1,62 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['relatedContainer'];
+
+    static values = {
+        index: Number,
+        label: String,
+        link: String,
+        deleteIcon: String,
+    };
+
+    connect() {
+        const container = this.element;
+        container
+            .querySelectorAll('.related-link-row')
+            .forEach((item) => {
+                this.#addButtonDeleteLink(item);
+            });
+    }
+
+    addRelatedElement() {
+        const rowNode = document.createElement('div');
+        rowNode.className = 'related-link-row';
+
+        const nodeLabel = this.#htmlToNode(this.labelValue.replace(
+            /__name__/g,
+            this.indexValue,
+        ));
+        rowNode.appendChild(nodeLabel);
+
+        const nodeLink = this.#htmlToNode(this.linkValue.replace(
+            /__name__/g,
+            this.indexValue,
+        ));
+        rowNode.appendChild(nodeLink);
+
+        this.#addButtonDeleteLink(rowNode);
+
+        this.relatedContainerTarget.appendChild(rowNode);
+        this.indexValue++;
+    }
+
+    #addButtonDeleteLink(item) {
```

Comment:
What is this hashtag for?

Reply:
Define a method private

![imagen](https://github.com/user-attachments/assets/61662133-4ccb-4175-a518-3247b1c40b8f)


https://railsdesigner.com/proper-stimulus-controllers/

Reply:
Today I learned... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties